### ISSUE TITLE
Does not work now

### ISSUE THREAD

**Issue by ParasiteEve0** [InvoxiPlayGames/FestivalInstrumentMapper#121]
After the latest update, it stopped working.

**Comment by faucetsoftware** [comment]
Yes the guitar didn't do anything then I press xbox button and it crashed.

**Comment by ParasiteEve0** [comment]
> After the latest update, it stopped working.

It now does?? Really weird.

**Comment by TheCreamSoda** [comment]
Same here, using a 360 XPlorer (or however you spell that). Using the controller hiding, restarted pc multiple times and made sure settings in game were correct. Game won't recognize the controller unless the hiding method is turned off, and when that is turned off it goes to the spinning around in circles issue that is well known at this point.

**Comment by Lesiu442** [comment]
Same for me, my PS4 Stratocaster doesn't do anything in game :(
